### DISCUSSION TITLE
Implementation of Cycle Charging (Feature Previously Written for CLOVER 4.0)

### DISCUSSION THREAD

**Discussion by hamishbeath** [CLOVER-energy/CLOVER#185] [IAA]
### Issue
To simulate hybrid mini-grid systems that have diesel configured so that the diesel genset can charge the batteries significant changes to the code are needed. These have already been implemented in a previous version of CLOVER.

### Proposal
To follow the same method and import the CLOVER 4.0 code into CLOVER 5.0. 

**Describe alternatives you've considered**
Leaving this feature out of CLOVER 5.0.

**Comment by BenWinchester** [comment]
Hi @hamishbeath , 
Does #22 depend on this/is it part of the same issue? If so, we can mark it as duplicate :smile:
